synapse/models/inet.py

```diff
@@ -1588,6 +1599,10 @@ def getModelDefs(self):
                         'interfaces': ('meta:taxonomy',),
                         'doc': 'A message type taxonomy.'}),
 
+                    ('inet:service:emote', ('guid', {}), {
```

Comment:
Since the emote is account specific, should this be 'inet:service:account:emote'?